lighthouse-core/report/v2/renderer/category-renderer.js

```diff
@@ -162,16 +162,10 @@ class CategoryRenderer {
   }
 
   /**
-   * @param {!Array<!ReportRenderer.AuditJSON>} audits
    * @param {!ReportRenderer.GroupJSON} group
-   * @param {function(!ReportRenderer.AuditJSON): !Element=} renderAudit
    * @return {!Element}
    */
-  _renderAuditGroup(audits, group, renderAudit) {
-    if (!renderAudit) {
-      renderAudit = this._renderAudit.bind(this);
-    }
-
+  _renderAuditGroup(group) {
```

Comment:
maybe add a jsdoc description saying something about audits to be put in the group can be appended to the returned element?